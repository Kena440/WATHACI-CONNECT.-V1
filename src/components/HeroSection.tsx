import { Button } from '@/components/ui/button';
import { ArrowRight, Users, TrendingUp, ShieldCheck } from 'lucide-react';
import { Link } from 'react-router-dom';

const HeroSection = () => {
  return (
    <section
      className="relative py-20 bg-center bg-cover"
      style={{
        backgroundImage:
          "url('/images/ChatGPT%20Image%20Sep%2014%2C%202025%2C%2011_09_30%20PM.png')",
      }}
    >
      <div className="absolute inset-0 bg-gradient-to-br from-orange-50/90 via-white/90 to-green-50/90" />
      <div className="relative max-w-6xl mx-auto px-6">
        <div className="text-center mb-16">
          <div className="mb-8">
            <img
              src="https://d64gsuwffb70l.cloudfront.net/686a39ec793daf0c658a746a_1753699300137_a4fb9790.png"
              alt="WATHACI CONNECT"
              loading="lazy"
              decoding="async"
              className="h-32 w-auto mx-auto drop-shadow-2xl"
            />
          </div>
          <h1 className="text-5xl font-bold text-gray-900 mb-6 leading-tight">
            Empowering Zambian
            <span className="text-transparent bg-clip-text bg-gradient-to-r from-orange-600 to-green-600"> Business Excellence</span>
          </h1>
          <p className="text-xl text-gray-600 mb-8 max-w-3xl mx-auto leading-relaxed">
            Connect with professional services, find skilled freelancers, and access resources 
            designed specifically for Zambian businesses. Your gateway to growth and success.
          </p>
          <div className="flex flex-col sm:flex-row gap-4 justify-center">
            <Link to="/get-started">
              <Button size="lg" className="bg-gradient-to-r from-orange-600 to-orange-700 hover:from-orange-700 hover:to-orange-800 text-white px-8 py-3 text-lg">
                Get Started Today
                <ArrowRight className="ml-2 w-5 h-5" />
              </Button>
            </Link>
            <Link to="/marketplace">
              <Button variant="outline" size="lg" className="border-2 border-green-600 text-green-700 hover:bg-green-50 px-8 py-3 text-lg">
                Explore Marketplace
              </Button>
            </Link>
          </div>
        </div>

        <div className="grid md:grid-cols-3 gap-8 mt-16">
          <div
            role="button"
            tabIndex={0}
            data-analytics-id="professional-network"
            aria-label="Join the professional network"
            title="Expand your professional contacts"
            className="text-center p-6 bg-white rounded-xl shadow-lg border border-blue-100 hover:shadow-xl focus:shadow-xl transition-shadow focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-blue-500"
          >
            <div className="w-16 h-16 bg-gradient-to-br from-blue-500 to-blue-600 rounded-full flex items-center justify-center mx-auto mb-4">
              <Users aria-hidden="true" className="w-8 h-8 text-white" />
            </div>
            <h3 className="text-xl font-semibold text-gray-900 mb-2">Connect with Pros</h3>
            <p className="text-gray-600">
              Connect with verified professionals and service providers across Zambia
            </p>
          </div>

          <div
            role="button"
            tabIndex={0}
            data-analytics-id="business-growth"
            aria-label="Grow your business"
            title="Access tools to accelerate your business"
            className="text-center p-6 bg-white rounded-xl shadow-lg border border-green-100 hover:shadow-xl focus:shadow-xl transition-shadow focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-green-500"
          >
            <div className="w-16 h-16 bg-gradient-to-br from-green-500 to-green-600 rounded-full flex items-center justify-center mx-auto mb-4">
              <TrendingUp aria-hidden="true" className="w-8 h-8 text-white" />
            </div>
            <h3 className="text-xl font-semibold text-gray-900 mb-2">Grow Your Business</h3>
            <p className="text-gray-600">
              Access tools and resources designed to accelerate your business growth
            </p>
          </div>

          <div
            role="button"
            tabIndex={0}
            data-analytics-id="quality-assurance"
            aria-label="Ensure quality"
            title="Work with vetted, reliable professionals"
            className="text-center p-6 bg-white rounded-xl shadow-lg border border-amber-100 hover:shadow-xl focus:shadow-xl transition-shadow focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-amber-500"
          >
            <div className="w-16 h-16 bg-gradient-to-br from-amber-500 to-amber-600 rounded-full flex items-center justify-center mx-auto mb-4">
              <ShieldCheck aria-hidden="true" className="w-8 h-8 text-white" />
            </div>
            <h3 className="text-xl font-semibold text-gray-900 mb-2">Ensure Quality</h3>
            <p className="text-gray-600">
              All services and professionals are vetted for quality and reliability
            </p>
          </div>
        </div>
      </div>
    </section>
  );
};

export { HeroSection };
export default HeroSection;