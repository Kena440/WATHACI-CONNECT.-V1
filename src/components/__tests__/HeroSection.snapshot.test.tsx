import { render } from '@testing-library/react';
import { MemoryRouter } from 'react-router-dom';
import { describe, it, expect } from 'vitest';
import HeroSection from '../HeroSection';

describe('HeroSection', () => {
  it('matches snapshot', () => {
    const { container } = render(
      <MemoryRouter>
        <HeroSection />
      </MemoryRouter>
    );
    expect(container).toMatchSnapshot();
  });
});
