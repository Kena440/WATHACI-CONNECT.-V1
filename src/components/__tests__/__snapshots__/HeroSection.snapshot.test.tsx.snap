// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`HeroSection > matches snapshot 1`] = `
<div>
  <section
    class="bg-gradient-to-br from-orange-50 via-white to-green-50 py-20"
  >
    <div
      class="max-w-6xl mx-auto px-6"
    >
      <div
        class="text-center mb-16"
      >
        <div
          class="mb-8"
        >
          <img
            alt="WATHACI CONNECT"
            class="h-32 w-auto mx-auto drop-shadow-2xl"
            src="https://d64gsuwffb70l.cloudfront.net/686a39ec793daf0c658a746a_1753699300137_a4fb9790.png"
          />
        </div>
        <h1
          class="text-5xl font-bold text-gray-900 mb-6 leading-tight"
        >
          Empowering Zambian
          <span
            class="text-transparent bg-clip-text bg-gradient-to-r from-orange-600 to-green-600"
          >
             Business Excellence
          </span>
        </h1>
        <p
          class="text-xl text-gray-600 mb-8 max-w-3xl mx-auto leading-relaxed"
        >
          Connect with professional services, find skilled freelancers, and access resources designed specifically for Zambian businesses. Your gateway to growth and success.
        </p>
        <div
          class="flex flex-col sm:flex-row gap-4 justify-center"
        >
          <a
            href="/get-started"
          >
            <button
              class="inline-flex items-center justify-center whitespace-nowrap font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-primary hover:bg-primary/90 h-11 rounded-md bg-gradient-to-r from-orange-600 to-orange-700 hover:from-orange-700 hover:to-orange-800 text-white px-8 py-3 text-lg"
            >
              Get Started Today
              <svg
                class="lucide lucide-arrow-right ml-2 w-5 h-5"
                fill="none"
                height="24"
                stroke="currentColor"
                stroke-linecap="round"
                stroke-linejoin="round"
                stroke-width="2"
                viewBox="0 0 24 24"
                width="24"
                xmlns="http://www.w3.org/2000/svg"
              >
                <path
                  d="M5 12h14"
                />
                <path
                  d="m12 5 7 7-7 7"
                />
              </svg>
            </button>
          </a>
          <a
            href="/marketplace"
          >
            <button
              class="inline-flex items-center justify-center whitespace-nowrap font-medium ring-offset-background transition-colors focus-visible:outline-none focus-visible:ring-2 focus-visible:ring-ring focus-visible:ring-offset-2 disabled:pointer-events-none disabled:opacity-50 bg-background hover:text-accent-foreground h-11 rounded-md border-2 border-green-600 text-green-700 hover:bg-green-50 px-8 py-3 text-lg"
            >
              Explore Marketplace
            </button>
          </a>
        </div>
      </div>
      <div
        class="grid md:grid-cols-3 gap-8 mt-16"
      >
        <div
          class="text-center p-6 bg-white rounded-xl shadow-lg border border-orange-100 hover:shadow-xl transition-shadow"
        >
          <div
            class="w-16 h-16 bg-gradient-to-br from-orange-500 to-orange-600 rounded-full flex items-center justify-center mx-auto mb-4"
          >
            <svg
              class="lucide lucide-users w-8 h-8 text-white"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="M16 21v-2a4 4 0 0 0-4-4H6a4 4 0 0 0-4 4v2"
              />
              <circle
                cx="9"
                cy="7"
                r="4"
              />
              <path
                d="M22 21v-2a4 4 0 0 0-3-3.87"
              />
              <path
                d="M16 3.13a4 4 0 0 1 0 7.75"
              />
            </svg>
          </div>
          <h3
            class="text-xl font-semibold text-gray-900 mb-2"
          >
            Professional Network
          </h3>
          <p
            class="text-gray-600"
          >
            Connect with verified professionals and service providers across Zambia
          </p>
        </div>
        <div
          class="text-center p-6 bg-white rounded-xl shadow-lg border border-green-100 hover:shadow-xl transition-shadow"
        >
          <div
            class="w-16 h-16 bg-gradient-to-br from-green-500 to-green-600 rounded-full flex items-center justify-center mx-auto mb-4"
          >
            <svg
              class="lucide lucide-trending-up w-8 h-8 text-white"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <polyline
                points="22 7 13.5 15.5 8.5 10.5 2 17"
              />
              <polyline
                points="16 7 22 7 22 13"
              />
            </svg>
          </div>
          <h3
            class="text-xl font-semibold text-gray-900 mb-2"
          >
            Business Growth
          </h3>
          <p
            class="text-gray-600"
          >
            Access tools and resources designed to accelerate your business growth
          </p>
        </div>
        <div
          class="text-center p-6 bg-white rounded-xl shadow-lg border border-orange-100 hover:shadow-xl transition-shadow"
        >
          <div
            class="w-16 h-16 bg-gradient-to-br from-orange-500 to-green-500 rounded-full flex items-center justify-center mx-auto mb-4"
          >
            <svg
              class="lucide lucide-award w-8 h-8 text-white"
              fill="none"
              height="24"
              stroke="currentColor"
              stroke-linecap="round"
              stroke-linejoin="round"
              stroke-width="2"
              viewBox="0 0 24 24"
              width="24"
              xmlns="http://www.w3.org/2000/svg"
            >
              <path
                d="m15.477 12.89 1.515 8.526a.5.5 0 0 1-.81.47l-3.58-2.687a1 1 0 0 0-1.197 0l-3.586 2.686a.5.5 0 0 1-.81-.469l1.514-8.526"
              />
              <circle
                cx="12"
                cy="8"
                r="6"
              />
            </svg>
          </div>
          <h3
            class="text-xl font-semibold text-gray-900 mb-2"
          >
            Quality Assurance
          </h3>
          <p
            class="text-gray-600"
          >
            All services and professionals are vetted for quality and reliability
          </p>
        </div>
      </div>
    </div>
  </section>
</div>
`;
